src/test/java/com/fasterxml/jackson/databind/struct/ShapeArrayWithAnyGetter4961Test.java

```diff
@@ -21,7 +20,6 @@ static class WrapperForAnyGetter {
         public BeanWithAnyGetter value;
     }
 
-    @JsonPropertyOrder({ "firstProperty", "secondProperties", "anyProperty", "forthProperty" })
```

Comment:
This should not have been removed because ordering for 3.0 will differ from 2.x. Will add, except for "anyProperty"